lua/sniprun.lua

```diff
@@ -42,7 +52,7 @@ function M.load_vimscript_config()
   vimscript_config["repl_disable"] = vim.g.SnipRun_repl_behavior_disable or M.config_values["repl_disable"]
   vimscript_config["selected_interpreters"] = vim.g.SnipRun_select_interpreters or M.config_values["selected_interpreters"]
   vimscript_config["inline_messages"] = vim.g.SnipRun_inline_messages or M.config_values["inline_messages"]
-
+  vimscript_config["SniprunColors"] = vim.g.SnipRunColors or M.config_values["SniprunColors"]
```

Comment:
not necessary: the vimscript config is deprecated and only exists to provide backward compatibility with neovim < 0.5.

For those installs, sniprun won't be updated to any version > 0.4.9 (which will contain your contribution)

Reply:
I will remove this line .